Src/Plugins/Security/SEC101_051.GpgCredentialValidator.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Utilities;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
+{
+    public class GpgCredentialValidator : ValidatorBase
+    {
+        internal static GpgCredentialValidator Instance;
+
+        static GpgCredentialValidator()
+        {
+            Instance = new GpgCredentialValidator();
+        }
+
+        public static string IsValidStatic(ref string matchedPattern,
+                                           ref Dictionary<string, string> groups,
+                                           ref string failureLevel,
+                                           ref string fingerprint,
+                                           ref string message)
+        {
+            return IsValidStatic(Instance,
+                                 ref matchedPattern,
+                                 ref groups,
+                                 ref failureLevel,
+                                 ref fingerprint,
+                                 ref message);
+        }
+
+        public static string IsValidDynamic(ref string fingerprint, ref string message)
```

Comment:
remove this

Reply:
Done

---
In reply to: [585980602](https://github.com/microsoft/sarif-pattern-matcher/pull/249#discussion_r585980602) [](ancestors = 585980602)